src/modules/AdminPage/components/ContentWrapper/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useState, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { AdminPageContentWrapper, ListTitle, CourseListSettings } from './styled';
+import { HeaderDecor } from 'modules/TeamsList/components/Teams/components/MyTeam/styled';
+import { AddCourseBlock } from './components/AddCourseBlock';
+import { CoursesList } from './components/CoursesList';
+import { useCoursesQuery } from 'hooks/graphql';
+import { ErrorModal, Loader } from 'components';
+import { Course } from 'types';
+import { ShowCourseSelect } from './components/ShowCourseSelect';
+import { ModalInput } from 'typography';
+import { SHOW_COURSES_OPTIONS } from 'appConstants';
+
+const filterCourseList = (courses: Course[], filter: string, searchValue?: string) => {
+  switch (filter) {
+    case 'Active':
+      return courses.filter(
+        ({ isActive, name }) => isActive && (searchValue ? name.includes(searchValue) : true)
+      );
+    case 'Terminated':
+      return courses.filter(
+        ({ isActive, name }) => !isActive && (searchValue ? name.includes(searchValue) : true)
+      );
+    case 'All':
+    default:
+      return searchValue ? courses.filter(({ name }) => name.includes(searchValue)) : courses;
+  }
+};
+
+export const AdminPageWrapper: FC = () => {
+  const { t } = useTranslation();
+  const { loading, courses, error } = useCoursesQuery();
+  const [currentShowCoursesOption, setCurrentShowCoursesOption] = useState(
+    SHOW_COURSES_OPTIONS[0].name
+  );
+  const [searchValue, setSearchValue] = useState('');
+
+  const isCourseNameUniq = (courseName: string) => {
```

Comment:
names of the **functions** should start with verb like _go, make, shake, set_ and so on.
we start generally **boolean** variables with **is**

Reply:
fixed